packages/lit-html/src/directives/async-replace.ts

```diff
@@ -36,43 +53,49 @@ class AsyncReplaceDirective extends AsyncDirective {
   // because its return value must be `noChange`
   private async __iterate(mapper?: Mapper<unknown>) {
```

Comment:
Nit: No longer async. If you want to return a promise still, we could `return forAwaitOf(…)` instead.
```suggestion
  private __iterate(mapper?: Mapper<unknown>) {
```

Reply:
Good catch, it can be inlined back into update now.
Done.